### PR TITLE
@fiery06 JAMES-3405 Update Prometheus config document

### PR DESCRIPTION
**Pull request by fiery06**

### CODE REVIEW

File: docs/modules/servers/pages/distributed/operate/metrics.adoc [hunk 1]
```diff
@@ -4,12 +4,22 @@
...
+    scrape_interval: 30s
+    metrics_path: /metrics
+    static_configs:
+      - targets: ['james:8000']
+....
 
-You can also export the metrics to ElasticSearch and visualize them with https://grafana.com/[Grafana].
-See xref:distributed/configure/elasticsearch.adoc#_exporting_metrics_directly_to_elasticsearch[elaticseach.properties]
-documentation for more details on how to set this up.
+You can also export the metrics to ElasticSearch and visualize them in https://grafana.com/[Grafana] dashboards.
```
Comment (Arsnael) [inline docs/modules/servers/pages/distributed/operate/metrics.adoc:22]: Why changing this part?

File: docs/modules/servers/pages/distributed/operate/metrics.adoc [hunk 2]
```diff
@@ -95,25 +105,35 @@ This is for instance how the JMAP dashboard looks like:
 
 image::metrics.png[metrics for the JMAP protocol request latencies]
 
-== Running and configuring Grafana
+== Running and configuring Prometheus/Grafana
```
Comment (Arsnael) [inline docs/modules/servers/pages/distributed/operate/metrics.adoc:108]: I think you should leave this section intact, as it is describing how to run Grafana with the ES metric exporter solution, that some people might still use, and create a new section for Prometheus/Grafana

File: docs/modules/servers/pages/distributed/operate/metrics.adoc [hunk 3]
```diff
@@ -95,25 +105,35 @@ This is for instance how the JMAP dashboard looks like:
...
+    #  - ./conf/grafana/grafana.ini:/etc/grafana/grafana.ini
+    #  - ./conf/grafana/provisioning:/etc/grafana/provisioning
+
+  prometheus:
+    image: prom/prometheus:latest
+    restart: unless-stopped
+    ports:
+      - "9090:9090"
+    volumes: #Map your scrape config to prometheus container.
+      - ./conf/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
```
Comment (chibenwa) [inline docs/modules/servers/pages/distributed/operate/metrics.adoc:131]: Can you include sample prometheus.yml as well as an example?

File: docs/modules/servers/pages/distributed/operate/metrics.adoc (hunk 1 above)
Comment (fiery06) [reply docs/modules/servers/pages/distributed/operate/metrics.adoc:22]: Ok I can put the ElasticSearch/Grafana section back

File: docs/modules/servers/pages/distributed/operate/metrics.adoc (hunk 3 above)
Comment (fiery06) [reply docs/modules/servers/pages/distributed/operate/metrics.adoc:131]: sure, no problem with that.

**Pull request closed without merging**